src/app/event/event-service.js

```diff
@@ -100,7 +100,11 @@ function EventService($http, $q, $rootScope, EventFactory) {
     };
 
     this.isDj = function() {
```

Comment:
this method is no longer necessary within the event-service, because it is now in the rootscope method